explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/v2/RawConverter.java

```diff
@@ -0,0 +1,99 @@
+package org.kie.trustyai.connectors.kserve.v2;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.protobuf.ByteString;
+
+public class RawConverter {
+
+    public static List<Boolean> toBoolean(ByteString raw) {
+        byte[] bytes = raw.toByteArray();
+        final List<Boolean> booleanList = new ArrayList<>();
+        for (byte b : bytes) {
+            for (int i = 7; i >= 0; --i) {
+                booleanList.add((b & (1 << i)) != 0);
+            }
+        }
+        return booleanList;
+    }
+
+    public static List<Integer> toInteger(ByteString raw) {
```

Comment:
This parses `int`s backwards: 1 is being encoded to `72057594037927936`. I think the encoding is little-endian, at least for integers.